lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -569,50 +573,54 @@ static void AddRequiredAliases(Block *block, lldb::StackFrameSP &stack_frame_sp,
 
   if (!self_type.IsValid() ||
       !self_type.GetTypeSystem()->SupportsLanguage(lldb::eLanguageTypeSwift))
-    return;
+    return Status("Could not add required aliases: self is not valid.");
 
   // Import before getting the unbound version, because the unbound
   // version may not be in the mangled name map.
 
   CompilerType imported_self_type = ImportType(swift_ast_context, self_type);
 
   if (!imported_self_type.IsValid())
-    return;
+    return Status(
+        "Could not add required aliases: imported self type is not valid.");
```

Comment:
```suggestion
        "Unable to add the aliases the expression needs because the `self` type from an import isn't valid.");
```

If the word "Unable" isn't a valid option, then I suggest the contraction "Couldn't".